roles/language/tasks/main.yml

```diff
@@ -1,9 +1,13 @@
 ---
-- name: install language package(s)
-  sudo: yes
-  apt: pkg={{ item }} state=latest force=yes
-  with_items: language_packages
+#- name: install language package(s)
+#  sudo: yes
+#  apt: pkg={{ item }} state=latest force=yes
+#  with_items: language_packages
 
+- name: generate locales
+  shell: "locale-gen {{ language }}"
+  sudo: yes
+  
```

Comment:
I was getting `invalid locale settings` error on provisioning (which I luckily [documented](http://www.mzoo.org/beyond-t1-micro-for-ofn_deployment/), since it's been since November that I've looked at any of this). I think this [SO](http://askubuntu.com/questions/162391/how-do-i-fix-my-locale-issue) posting may have lead me in this direction.
Local-gen man page explains:

> Compiled locale files take about 50MB of disk space, and most users only need few locales. In order to save disk space, compiled locale files are not distributed in the locales package, but selected locales are automatically generated when this package is installed by running the locale-gen program.